3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb

```diff
@@ -1141,11 +1149,18 @@ def self.bootstrap_failure?(jobflow, cluster_step_statuses)
           (!(jobflow.cluster_status.last_state_change_reason =~ bootstrap_failure_indicator).nil?)
       end
 
-      Contract String, String => Elasticity::CustomJarStep
-      def get_rmr_step(location, bucket)
+      Contract String, String, String => Elasticity::CustomJarStep
+      def get_rmr_step(location, bucket, description)
         step = Elasticity::CustomJarStep.new("s3://#{@jobflow.region}.elasticmapreduce/libs/script-runner/script-runner.jar")
-        step.arguments = ["#{bucket}common/emr/snowplow-hadoop-fs-rmr-0.1.0.sh", location]
-        step.name << ": Empty Raw HDFS"
+        step.arguments = ["#{bucket}common/emr/snowplow-hadoop-fs-rmr-0.2.0.sh", location]
+        step.name << ": #{description}"
+        step
+      end
+
+      def get_hdfs_expunge_step
+        step = Elasticity::CustomJarStep.new("command-runner.jar")
+        step.arguments = %W(hdfs dfs -expunge)
```

Comment:
does this work with `command-runner`? it's not listed in https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-commandrunner.html

Reply:
Tested and working @BenFradet - you can execute any bash command using `command-runner.jar` - the only reason I could not use it for all the removals was that I needed to add if statement checks.